Marlin/pins_ULTIMAKER_OLD.h

```diff
@@ -95,3 +95,7 @@
 #define LCD_PINS_D5        34
 #define LCD_PINS_D6        32
 #define LCD_PINS_D7        30
+
+//#define SPINDLE_ENABLE_PIN 44
+//#define SPINDLE_SPEED_PIN  45 // needs a hardware PWM
+//#define CASE_LIGHT_PIN     46 // needs a hardware PWM
```

Comment:
I think we need to be more authoritative on the pins defined for each board. The `E0_DIR_PIN` is pin 45 in this file, for example.

Also, the idea is that these pins are not commented out, but are explicitly defined for use by the enabled feature. In this case, the feature is `SPINDLE_ENABLE`, enabling any code that refers to the spindle, using these pins.

The same should really apply with the `CASE_LIGHT_PIN` too, but that's another topic.